community/php7/APKBUILD

```diff
@@ -1,71 +1,70 @@
 # Contributor: Valery Kartel <valery.kartel@gmail.com>
 # Maintainer: Valery Kartel <valery.kartel@gmail.com>
 pkgname=php7
-_pkgreal=php
 pkgver=7.0.16
-pkgrel=0
-pkgdesc="The PHP language runtime engine - 7th branch"
+pkgrel=1
+_apiver=20151012
+_suffix=${pkgname#php}
+_pkgreal=${pkgname%$_suffix}
+pkgdesc="The PHP$_suffix language runtime engine"
 url="http://www.php.net/"
 arch="all"
 license="PHP-3"
 options=""
-depends="$pkgname-common"
-depends_dom="$pkgname-xml"
-depends_mysqlnd="$pkgname-mysqli"
-depends_phar="$pkgname"
-depends_soap="$pkgname-xml"
-depends_wddx="$pkgname-xml"
-depends_xmlreader="$pkgname-dom"
-depends_xmlrpc="$pkgname-xml"
-depends_xsl="$pkgname-dom"
+depends="$pkgname-config"
+depends_dev="$pkgname"
+replaces="$_pkgreal $_pkgreal-cli"
+provides="$replaces $pkgname-cli"
 makedepends="autoconf bison re2c apache2-dev libxml2-dev libxslt-dev libzip-dev bzip2-dev zlib-dev
 	aspell-dev enchant-dev expat-dev pcre-dev curl-dev gmp-dev icu-dev imap-dev
 	libical-dev libressl-dev openldap-dev net-snmp-dev db-dev krb5-dev gdbm-dev sqlite-dev
 	freetds-dev mariadb-dev postgresql-dev unixodbc-dev freetype-dev tidyhtml-dev libxpm-dev
-	libpng-dev libwebp-dev libjpeg-turbo-dev libmcrypt-dev gsoap-dev recode-dev
-	libedit-dev paxmark gettext-dev
+	libpng-dev libwebp-dev libjpeg-turbo-dev libmcrypt-dev recode-dev libedit-dev gettext-dev
 	"
 source="http://php.net/distributions/$_pkgreal-$pkgver.tar.bz2
-	$pkgname-fpm.initd
-	$pkgname-fpm.logrotate
 	$pkgname-module.conf
-	install-pear.patch
-	tidy-buffio.patch
+	$pkgname-fpm.logrotate
+	$pkgname-fpm.initd
+	$pkgname-fpm.patch
 	includedir.patch
+	tidy-buffio.patch
+	install-pear.patch
 	fix-asm-constraints-in-aarch64-multiply-macro.patch
```

Comment:
No, it’s not in 7.1.4, see https://github.com/php/php-src/blob/php-7.1.4/Zend/zend_multiply.h#L56.